### PR TITLE
Awesome Books es6

### PR DESCRIPTION
**Pull request by omar25ahmed**
### ES6 modules and npm:
- Learned how to use ES6 modules to write modular JavaScript.
- Learned how to use proper ES6 syntax.
- I used npm as a software packages system.

### CODE REVIEW

File: JS/app.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
+/* eslint-disable max-classes-per-file */
+class Book {
```
Comment (Gambit142) [inline JS/app.js]: - [x] Nice work with the project. However, I noticed that you do not need this folder as you have the same logic in the _**modules/class.js**_. Kindly delete this folder as you are required to put all other logics apart from your _**index.js**_ in the module folder. 😄

File: modules/class.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(title, author) {
```
Comment (Gambit142) [inline modules/class.js]: - [x] I understand that you are struggling with `eslint` errors here, I would like to suggest that you transfer some _**related classes**_ to a new file in the modules folder. That way, you can separate concerns and your code is easy to maintain. 👍

File: index.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+document.querySelector('#list').addEventListener('click', (e) => {
+  showBooks.deleteBook(e.target);
+
+  Store.removeBook(e.target.parentElement.previousElementSibling.textContent);
+});
+
+document.getElementById('date').innerHTML = DateTime.now().toLocaleString(DateTime.TIME_SIMPLE);
+
+// const date = new Date();
+// document.getElementById('date').innerHTML = date;
```
Comment (Gambit142) [inline index.js]: - [x] Kindly remove all commented-out codes from your code base. It helps to keep the code clean.

File: modules/class.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
```
Comment (Gambit142) [inline modules/class.js]: - [ ](Optional) I can see that you are repeating this code severally in this file,=. I would like to suggest that you put it a function and call it wherever you need it to be called. The same suggestion goes for this line of code below:

`localStorage.setItem('books', JSON.stringify(books))`

File: index.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+function showAdd() {
+  document.getElementById('book-list').style.display = 'none';
+  document.getElementById('add-new').style.display = 'flex';
+  document.getElementById('contact').style.display = 'none';
+}
+function showContactPage() {
+  document.getElementById('book-list').style.display = 'none';
+  document.getElementById('add-new').style.display = 'none';
+  document.getElementById('contact').style.display = 'flex';
+}
```
Comment (Gambit142) [inline index.js]: - [x] Kindly use `ES6` syntax for creating a function. It looks like this `const showContactPage = () => {some code}`

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @omar25ahmed 👋, 

Good job so far!

Highlight

- Good use of modular javascript ✔️ 
- The app works as expected ✔️ 

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider the optional suggestion I made also.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @omar25ahmed , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**